File: src/Asset/Vns.Erp.Core.Asset.CsAssetGUI/Asset/PTKT.cs
using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Diagnostics;
using Vns.Erp.Core.Asset.Service.Interface;
using Vns.Erp.Core.Asset.Domain;

using Vns.Erp.Core;

namespace Vns.Erp.Core.Asset.CsAssetGUI
{
    public partial class PTKT
    {
        #region "Property"
        private ITsTscdPtungService _TsTscdPtungService;
        public ITsTscdPtungService TsTscdPtungService
        {
            get { return _TsTscdPtungService; }
            set { _TsTscdPtungService = value; }
        }
        #endregion

        public DataTable GetData(Guid TSCD_ID)
        {
            DataTable dt = new DataTable();
            dt = DataHelper.ToDataTable<TsTscdPtung>(_TsTscdPtungService.GetAllByTSCD_ID(TSCD_ID));
            return dt;
        }

    }
}

File: src/Asset/Vns.Erp.Core.Asset.VbAssetGUI/PTKT.vb
Imports Vns.Erp.Core.Asset.Service.Interface
Imports Vns.Erp.Core.Asset.Domain

Public Class PTKT
#Region "Property"
    Private _TsTscdPtungService As ITsTscdPtungService
    Public Property TsTscdPtungService As ITsTscdPtungService
        Get
            Return _TsTscdPtungService
        End Get
        Set(ByVal value As ITsTscdPtungService)
            _TsTscdPtungService = value
        End Set
    End Property
#End Region

    Public Function GetData(ByVal TSCD_ID As Guid) As DataTable
        Dim dt As DataTable = New DataTable
        dt = DataHelper.ToDataTable(Of TsTscdPtung)(_TsTscdPtungService.GetAllByTSCD_ID(TSCD_ID))
        Return dt
    End Function

End Class